modules/enabling-data-science-pipelines-2.adoc

```diff
@@ -14,61 +14,103 @@ endif::[]
 ifdef::cloud-service[]
 Data science pipelines in {productname-short} are now based on link:https://www.kubeflow.org/docs/components/pipelines/v2/[KubeFlow Pipelines (KFP) version 2.0]. DSP 2.0 is enabled and deployed by default in {productname-short}.
 endif::[]
+endif::[]
+
+[NOTE]
+====
+The `PipelineConf` class is deprecated, and there is no KFP 2.0 equivalent.
+====
 
 [IMPORTANT]
 ====
 Data Science Pipelines (DSP) 2.0 contains an installation of Argo Workflows. {productname-short} does not support direct customer usage of this installation of Argo Workflows.
 
 ifdef::upstream[]
-If there is an existing installation of Argo Workflows that is not installed by DSP on your cluster, installations of and upgrades to {productname-short} 2.10.0 with DSP will fail.
-
-To install or upgrade to {productname-short} 2.10.0 with DSP, ensure that no separate installation of Argo Workflows exists on your cluster.
+To install or upgrade to {productname-short} 2.10.0 with DSP, ensure that no installation of Argo Workflows that is not installed by {productname-short} exists on your cluster.
```

Comment:
Suggest:
_, ensure that your cluster does not have an existing installation of Argo Workflows that is not installed by {productname-short}._

Reply:
Updated.